OrbitGl/App.cpp

```diff
@@ -126,8 +127,9 @@ PresetLoadState GetPresetLoadStateForProcess(
 
 bool DoZoom = false;
 
-OrbitApp::OrbitApp(MainThreadExecutor* main_thread_executor)
-    : main_thread_executor_(main_thread_executor) {
+OrbitApp::OrbitApp(MainThreadExecutor* main_thread_executor,
+                   orbit_metrics_uploader::MetricsUploader* metrics_uploader)
+    : main_thread_executor_(main_thread_executor), metrics_uploader_(metrics_uploader) {
```

Comment:
Could you put all the integration changes in OrbitGl and OrbitQt into a separate commit?

Reply:
Done